src/main/java/org/folio/circulation/resources/RequestFromRepresentationService.java

```diff
@@ -272,16 +280,42 @@ private CompletableFuture<Result<Request>> fetchItemAndLoanForRecallTlrRequest(R
         requestQueue.getRecalledLoansIds())
```

Comment:
Extract this as a variable and pass into shouldCheckRecalledLoan

Reply:
It is extracted, thanks.